tests/leagueResource.spec.js

```diff
@@ -46,16 +46,16 @@ describe ("resource: leagueResource", function() {
 
   // building urls
   beforeEach(function() {
-    spyOn(yf, "api").andCallThrough();
+    spyOn(yf, "api").and.callThrough();
   });
 
   // meta
   it ("should build a proper url to retrieve metadata via a league key", function() {
     nock('https://fantasysports.yahooapis.com')
       .get("/fantasy/v2/league/328.l.34014/metadata?format=json")
-      .reply(200, {});
+      .reply(200, require("./nock-data/leagueMeta").meta);
```

Comment:
Need to return some data, because the object mapping classes expect there to be at least a fantasy_content field.